core/storage/suggestion/gae_models_test.py

```diff
@@ -3589,13 +3653,80 @@ def test_fetch_page_with_filtering(self) -> None:
                 offset=0,
                 sort_by=None,
                 topic_ids=None,
+                max_days_since_first_activity=None,
                 max_days_since_last_activity=7,
                 language_code='hi'
             ))
         self.assertEqual(len(sorted_results), 0)
         self.assertFalse(more)
         self.assertEqual(next_offset, 1)
 
+        # Check for no activity before last 8 days.
```

Comment:
These backend tests span way too many lines and are unfocused to the point where I don't understand why we are performing each check. I'm going to file a separate issue to refactor this test file. 

@masterboy376 Don't worry about this concern for now. Just focus on addressing all other reviewer comments and getting this PR merged. Thanks for your work!